TestResultSummaryService/perf/DataManagerAggregate.js

```diff
@@ -5,11 +5,10 @@ const ObjectID = require( 'mongodb' ).ObjectID;
 class DataManagerAggregate {
     static aggDataCollect(childBuild) {
         const benchmarkMetricsCollection = {};
-        let name, variant, jdkBuildDate;
+        let name, variant;
         if (Array.isArray(childBuild.tests) && childBuild.tests.length > 0 ) {
             name = childBuild.tests[0].benchmarkName;
```

Comment:
Currently, we assume that we could only have one test in a Jenkins job, but in future, when we add support for Adopt TKG builds, we'll need to add support for parsing all tests.